domains/phantomdev.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "siddddss",
+        "email": "discordacoforsid@gmail.com"
+    },
+    "record": {
+        "A": ["129.213.82.186"]
```

Comment:
Yeah no, this PR is going to be denied. Bunch of scam ads popped up, as well endless redirects.